README.adoc

```diff
@@ -127,6 +127,26 @@ public class FaultToleranceBean {
 }
 }
 ```
+##### Timeout Configs
+Since v2.1 we can configure the existing test timeouts. With the following configurations, timeouts can be greatly 
+reduced for local usage or in fast CI environments. 
+
+The properties can be set either with MP Config or System Properties. 
+
+.TCK Timeout configs
+|===
+|Property |Default |unit
+
+|org.eclipse.microprofile.fault.tolerance.basetimeout
+|100
+|ms
```

Comment:
We discussed this on today's call and decided not to introduce this property as the multiplier is efficient.

Reply:
Add instruction to running_the_tck doc.

Reply:
done